android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -137,6 +139,13 @@ public void releaseViews()
 	@Override
 	public void release()
 	{
+		// Release search bar proxy if there is one
+		if (getTableView().getNativeView() instanceof RelativeLayout) {
```

Comment:
Isn't all of this redundant with TiUITableView.java.release()? Why do we need to release the search view in both? Doesn't one "control" the flow and call release on the other one?